### DISCUSSION TITLE
How to define a new Preset in PrimeVue v.4

### DISCUSSION THREAD

**Discussion by fabxcp** [primefaces/community#2657] [PrimeVue]
Hello,
looking at the "Styled mode" documentation, I noticed that it's not recommended to override the component design tokens of a built-in preset like Aura (i.e. it's explained at this paragraph: https://primevue.org/theming/styled/#component). 
So, building my own preset should be the recommended approach in this case.
Could you please share some documentation or examples on how it's possible to define custom presets?

Thanks,
Fabio

**Comment by qwadrox** [comment]
hi @fabxcp! 

The docs says: 

> "Overriding components tokens is not the recommended approach if you are building your own style..."

The key part from that sentence is: **if you are building your own style**.
It means if you have a unique requirement, a design system, that does not fit at all into the existing presets, it is better to create a new preset. 
It would take much more time, so it's not recommended for every scenario.  

Although I don't know, is there an example how to create your own preset, there is this video on how to do the customization: https://www.youtube.com/watch?v=J3KFw5sih98

It might help you.
